### PR TITLE
[test] Integrate an existing testcase upon build by Catkin.

### PR DESCRIPTION
**Pull request by 130s**
There's a testcase in `rqt_tf_tree/test` folder but seems to have not been integrated.

I'm not entirely sure about d9f53ff but without this the following error was recorded:

```
 $ catkin_test_results
 build/rqt_tf_tree/test_results/rqt_tf_tree/nosetests-test.dotcode_tf_test.py.xml: 1 tests, 1 errors, 0 failures, 0 skipped
 Summary: 1 tests, 1 errors, 0 failures, 0 skipped
 $ more build/rqt_tf_tree/test_results/rqt_tf_tree/nosetests-test.dotcode_tf_test.py.xml
 <?xml version="1.0" encoding="UTF-8"?><testsuite name="nosetests" tests="1" errors="1" failures="0" skip="0"><testcase classname="nose.failure.Failure" name="runTest" time="0.000"><error type="exceptions.
ImportError" message="No module named rqt_tf_tree.dotcode_tf&#10;-------------------- &gt;&gt; begin captured logging &lt;&lt; --------------------&#10;rospy.topics: INFO: topicmanager initialized&#10;--------------------- &gt;&gt; end captured logging &lt;&lt; ---------------------"><![CDATA[  File "/usr/lib/python2.7/unittest/case.py", line 329, in run    testMethod()
      File "/usr/lib/python2.7/dist-packages/nose/loader.py", line 418, in loadTestsFromName
          addr.filename, addr.module)
	    File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 47, in importFromPath
	        return self.importFromDir(dir_path, fqname)
		  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 94, in importFromDir
		      mod = load_module(part_fqname, fh, filename, desc)
		        File "/home/n130s/link/ROS/indigo_trusty/cws_viz/src/ros-visualization/rqt_tf_tree/test/dotcode_tf_test.py", line 43, in <module>
    from rqt_tf_tree.dotcode_tf import RosTfTreeDotcodeGenerator
    'No module named rqt_tf_tree.dotcode_tf\n-------------------- >> begin captured logging << -----------
```

### CODE REVIEW

**Comment by 130s** [comment]
@Peter9606 would you be interested in reviewing this?

**Comment by Peter9606** [comment]
Sure, I'll give my comment in next 48 hours

**Comment by Peter9606** [comment]
Glad to see this patch made test passed.
There are one minor suggestion on the test code.
Seems the import for rospack and MagicMock is not necessary. I tried "catkin_make test" without importing those two, everything were okay.

File: test/dotcode_tf_test.py [hunk 1]
```diff
@@ -34,8 +34,7 @@
 import unittest
 import rospkg
```
Comment (Peter9606) [inline test/dotcode_tf_test.py]: rospkg isn't used anywhere, could be just deleted

File: test/dotcode_tf_test.py [hunk 2]
```diff
@@ -34,8 +34,7 @@
...
 
-import tf
-from tf.srv import *
+import tf2_ros
 
 # get mock from pypi as 'mock'
 from mock import Mock, MagicMock, patch
```
Comment (Peter9606) [inline test/dotcode_tf_test.py]: MagicMock isn't used anywhere, could be just deleted

Comment (Peter9606) [inline test/dotcode_tf_test.py]: @130s Is there any update?

Comment (Peter9606) [reply test/dotcode_tf_test.py]: @130s can you give an update here?

**Comment by 130s** [comment]
@Peter9606 Sorry for the delay. I hope I've addressed your comments.

**Comment by Peter9606** [comment]
@130 I'm not sure why Mpr__rqt_tf_tree__ubuntu_bionic_amd64 failed. Is it because of this PR is too old?

**Comment by dirk-thomas** [comment]
@ros-pull-request-builder retest this please

**Comment by 130s** [comment]
After re-running the test (thanks Dirk) all checks passed, so I assume it was either temporary glitch or something got corrected since the failure. Merging.

**Comment by Peter9606** [comment]
@dirk-thomas @130s  is it possible to grant me write permission to this project so I can do something that really helps.

**Comment by 130s** [comment]
tl;dr I couldn't add you @Peter9606. I think I don't have a right to do so.

I found `tf_tree_maintainers` where I'm added. I tried to add @Peter9606 but GitHub says that user is "not a member of this organization". Adding an account directly doesn't seem to be an option for me either.

Although not documented anywhere AFAIK, commonly in core packages in ROS merging PRs requires at least 1 approval. Everyone can give an approval regardless the account status. So at minimum you could help the maintenance in that way. For the particular PR https://github.com/ros-visualization/rqt_tf_tree/pull/5, I thought this got an approval so I merged but turned out it hasn't. So I'm sorry for hasty merge but hope that wasn't a bit deal this time.

**Comment by dirk-thomas** [comment]
@Peter9606 I have added you as a collaborator to the repo.

@130s I hope this was ok with you (since you said "I tried to add ..."). If not let me know I can revert it.
